presto-hive/src/main/java/com/facebook/presto/hive/s3/PrestoS3FileSystemStats.java

```diff
@@ -270,11 +272,27 @@ else if (t instanceof SocketTimeoutException) {
         else if (t instanceof AbortedException) {
             awsAbortedExceptions.update(1);
         }
+        else if (t instanceof S3Exception) {
+            otherReadErrors.update(1);
+        }
+        else if (t instanceof SdkClientException) {
+            otherReadErrors.update(1);
+        }
```

Comment:
nit: we can remove both of these conditions as they will be handled automatically in the final else block